### PR TITLE
Optimize request header

### PR DESCRIPTION
**Pull request by hoppenichu**

### CODE REVIEW

File: APIKit/Session.swift [hunk 1]
```diff
@@ -12,7 +12,9 @@ public class Session {
     public func sendRequest<T: RequestType>(request: T, handler: (Result<T.Response, APIError>) -> Void = {r in}) -> NSURLSessionDataTask? {
         switch request.buildURLRequest() {
         case .Failure(let error):
-            handler(.Failure(error))
+            dispatch_async(dispatch_get_main_queue(), { () -> Void in
+                handler(.Failure(error))
+            })
```
Comment (hoppenichu) [inline APIKit/Session.swift]: https://github.com/AFNetworking/AFNetworking/blob/master/AFNetworking/AFHTTPSessionManager.m#L267

Comment (ishkawa) [inline APIKit/Session.swift]: We prefer the following style:

```swift
dispatch_async(dispatch_get_main_queue()) {
    .Failure(error)
}
```

File: APIKit/RequestType.swift [hunk 2]
```diff
@@ -101,6 +101,7 @@ public extension RequestType {
         default:
             do {
                 URLRequest.HTTPBody = try requestBodyBuilder.buildBodyFromObject(parameters)
+                URLRequest.setValue(requestBodyBuilder.contentTypeHeader, forHTTPHeaderField: "Content-Type")
```
Comment (ishkawa) [inline APIKit/RequestType.swift]: :+1:

**Comment by ishkawa** [comment]
Except for 1 note, looks good :sparkles:

File: APIKit/Session.swift (hunk 1 above)
Comment (hoppenichu) [inline APIKit/Session.swift]: @ishkawa OK, I've fixed it.

**Comment by ishkawa** [comment]
All changes make sense. Thank you for the pull request!

**Comment by hoppenichu** [comment]
:beer:

**Pull request merged**